src/main/xerus/monstercat/api/Playlist.kt

```diff
@@ -40,14 +40,12 @@ object Playlist {
 		else -> nextSong()
 	}
 	
-	fun addNext(track: Track) = tracks.apply {
-		remove(track)
-		add(currentIndex.value?.let { it + 1 } ?: 0, track)
+	fun addNext(track: Track) {
+		tracks.apply { remove(track); add(currentIndex.value?.let { it + 1 } ?: 0, track) }
```

Comment:
Why are you using apply here? As I said, since it changes `this` it should only be used if there is a real justification.

Reply:
ed62aa36c31d0a84e079369cdb0bdf17bf6c3caf